.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+        env:
```

Comment:
I guess env should be top level of include element rather than under addons, considering examples in the doc.
https://docs.travis-ci.com/user/languages/c/

Reply:
Moved.